next-js/lotr-app/pages/volumes/index.js

```diff
@@ -0,0 +1,28 @@
+import Link from "next/link";
+import Head from "next/head";
+import { introduction } from "../../resources/lib/data.js";
+import { volumes } from "../../resources/lib/data.js";
+
+export default function Volumes() {
+  return (
+    <>
+      <Head>
+        <title>My LOTR App: Volumes</title>
+      </Head>
+      <h1>Lord of the Rings</h1>
+      <p>{introduction}</p>
+      <h2>All Volumes</h2>
+      <ul>
+        <li>
```

Comment:
you could map over all items in the volumes array and let it kick out links automatically